### ISSUE TITLE
[MODEL EVALUATION REQUEST] answerdotai/ModernBERT-base

### ISSUE THREAD

**Issue by timpal0l** [EuroEval/EuroEval#992]
### Model ID

answerdotai/ModernBERT-base

### Evaluation languages

- [x] Romance languages (French, Italian, Spanish)
- [x] Scandinavian languages (Danish, Faroese, Icelandic, Norwegian, Swedish)
- [x] West Germanic languages (Dutch, English, German)
- [x] Finnish

### Model type

Encoder model (e.g., BERT)

### Model size

Small (<=8B parameters)

### Merged model

Not a merged model

**Comment by saattrupdan** [comment]
Hi @timpal0l. Appreciate it, but the model is already on the leaderboards and was added in https://github.com/EuroEval/EuroEval/issues/672.

Also, just FYI for next time, please run the evaluation with the `--evaluate-test-split` flag to run it on the full test sets, rather than the (10x smaller) validation splits. It's not crucial, but provides more reliable results.

**Comment by saattrupdan** [comment]
I completely agree, and I have been confused about it as well. I tried digging into exactly what's going wrong, and it seems to be overfitting superfast on the datasets, resulting in bad scores. Even tried lowering the learning rate, but no change. There's really no special thing going on in the finetuning script, and it's consistent with the hyperparameters mentioned in the paper as well. It's really odd.
